hphp/zend/html_tables/generate-html-table.py

```diff
@@ -190,7 +190,7 @@ def make_multicode_table(file_name, table_name):
 output_mapping('mappings/KOI8-R.TXT', 'entity_table_cs_koi8r')
 #output_mapping('mappings/ROMAN.TXT', 'entity_table_cs_macroman')
 
-# table for encoding csjis is esentially empty
+# table for encoding cs-sjis is essentially empty
```

Comment:
`cs-sjis` is used elsewhere in the repository, and there didn't seem to be any particular value in this extra spelling.